spynnaker/pyNN/models/neuron/abstract_population_vertex.py

```diff
@@ -252,14 +251,26 @@ def mark_no_changes(self):
 
     def _get_buffered_sdram_per_timestep(self, vertex_slice):
```

Comment:
Not needed if get_buffered_sdram_per_timestep is changed to get_n_timesteps_in_buffer_space.

Reply:
See above on why get_n_timesteps_in_buffer_space is bad!